core/libs/commonwealth/commonwealth/utils/DHCPServerManager.py

```diff
@@ -101,16 +101,23 @@ def command_list(self) -> List[Union[str, pathlib.Path]]:
         ]
 
     def start(self) -> None:
```

Comment:
Just starting follow the project, congrats guys first of all. I am wondering if the attempts and others params could be loaded from config file and if not setup the config file or exists use the default value.

Reply:
Thanks, do you have any project running BlueOS or any idea of how to use it ? You can reach us via discord or bluerobotics forum if you wish.

Reply:
The default configuration are loaded via the main file, each project has its own settings.